model/config.go

```diff
@@ -123,6 +123,7 @@ type EmailSettings struct {
 	RequireEmailVerification bool
 	FeedbackName             string
 	FeedbackEmail            string
+	FeedbackOrganization     *string
```

Comment:
Shouldn't the default be `(c) 2015 Mattermost, Inc. 855 El Camino Real, 13A-168, Palo Alto, CA, 94301.` ??  maybe ask a PM.